spec/API_specification/data_type_functions.md

```diff
@@ -4,9 +4,72 @@
 
 A conforming implementation of the array API standard must provide and support the following data type functions.
 
+
 <!-- NOTE: please keep the constants in alphabetical order -->
 
 ## Objects in API
+(function-broadcast-arrays)=
+### broadcast_arrays(*args, /)
+
+Broadcast any number of arrays against each other.
+
+#### Parameters
+
+-   ***args**: _\[&lt;array&gt;\]_
+
+    -   arrays to broadcast.
+
+#### Returns
+
+-   **out**: _\[&lt;array&gt;\]_
+
+    -   a list of arrays where more than one element of a broadcasted array may refer to a single memory location.
+
+
+(function-broadcast-to)=
+### broadcast_to(array, /, shape)
+
+Broadcast an array to a new shape.
+
+-   A `ValueError` exception must be raised if the array is not compatible with the new shape according  to the semantics defined in {ref}`broadcasting`.
+
+#### Parameters
+
+-   **array**: _&lt;array&gt;_
+
+    -   array to broadcast.
+
+-   **shape**: Tuple[int, …]
+
+    -   shape of the desired array.
+
+#### Returns
+
+-   **out**: _&lt;array&gt;_
+
+    -   an array with the given shape where more than one element may refer to a single memory location.
+
+(function-can-cast)=
+### can_cast(from_, /, *, to)
```

Comment:
```suggestion
### can_cast(from, to, /)
```

`to` cannot be optional.

While `from` is a reserved keyword in Python, I don't believe we need to append a `_` suffix, as this argument is positional only.